NutritionManagement/src/main/User.java

```diff
@@ -2,20 +2,47 @@
 
 import java.util.*;
 
+import exceptions.ResourceNotFoundException;
+
 public class User {
 	private User() {
-		days=new ArrayList<>();
-		foodList=new ArrayList<>();
-		//WIP
+		days = new ArrayList<>();
+		foodList = new ArrayList<>();
 	}
-	
+
 	private static User instance = new User();
-	
+
 	public static User getInstance() {
 		return instance;
 	}
-	
+
 	private ArrayList<Day> days;
 	private ArrayList<Food> foodList;
-	
+
+	public void addFoodType(Food food) {
+		this.foodList.add(food);
+	}
+
+	public void deleteFoodType(Food food) {
+		this.foodList.remove(food);
+	}
+
+	public Food getFoodType(String name) throws ResourceNotFoundException {
```

Comment:
clearly communicating to the caller when a food item is not found in the list!

Reply:
btw, need to consider if we allow duplicated food name (I suggest no), and may need to handle it in `addFoodType`